packages/react-server/core/ClientController.js

```diff
@@ -55,6 +55,7 @@ class ClientController extends EventEmitter {
 
 		window.__reactServerTimingStart = window.performance ? window.performance.timing.navigationStart : new Date;
 		var wakeTime = new Date - window.__reactServerTimingStart;
+		var aboveTheFold = window.performance ? (window.__displayAboveTheFold - window.__reactServerTimingStart) : 0;
```

Comment:
yeah... I feel like this is also true for any values logged in ClientController that relies on `__reactServerTimingStart` , do you think we want to change those also?